pkg/search/postgres/store.go

```diff
@@ -293,3 +293,59 @@ func (s *GenericStore[T, PT]) GetIDs(ctx context.Context) ([]string, error) {
 
 	return identifiers, nil
 }
+
+// GetMany returns the objects specified by the IDs from the store as well as the index in the missing indices slice.
+func (s *GenericStore[T, PT]) GetMany(ctx context.Context, identifiers []string) ([]*T, []int, error) {
+	defer s.setPostgresOperationDurationTime(time.Now(), ops.GetMany)
+
+	if len(identifiers) == 0 {
+		return nil, nil, nil
+	}
+
+	var sacQueryFilter *v1.Query
+	if s.hasPermissionsChecker() {
+		if ok, err := s.permissionChecker.GetAllowed(ctx); err != nil {
+			return nil, nil, err
+		} else if !ok {
+			return nil, nil, sac.ErrResourceAccessDenied
```

Comment:
This changes the behaviour compared to the generated code.
I'd stick to the previous behaviour that returns `nil, nil, nil`, otherwise, this may leak information about the user allowed scope.